jordgubb/gfx.sy

```diff
@@ -0,0 +1,282 @@
+use love as l
+from love use love
+
+Color :: blob {
+    red: float,
+    green: float,
+    blue: float,
+    alpha: float,
+}
+
+// NOTE(ed): Might be nice to convert HSL to RGB?
+rgb :: fn r, g, b -> Color
+    Color { red: r, green: g, blue: b, alpha: 1. }
+end
+rgba :: fn r, g, b, a -> Color
+    Color { red: r, green: g, blue: b, alpha: a }
+end
+stableRandomColor :: fn i: float -> Color
+    r :: rem' 0.3 + i * 3.33, 1.
+    g :: rem' 0.2 + i * 7.77, 1.
+    b :: rem' 0.7 + i * 5.55, 1.
+    rgb' r, g, b
+end
+
+setColor :: fn color do
+    love.graphics.setColor' (color.red, color.green, color.blue, color.alpha)
+end
+
+drawLine :: fn color, a, b do
+    setColor' color
+    love.graphics.line' a[0], a[1], b[0], b[1]
+end
```

Comment:
patterns in parameters maybe? so we could write `fn color, (x0, y0), (x1, y1)`. not right now i realize but in the future

Reply:
Have you added it to *the list*?